docs/zope4/migration/zodb.rst

```diff
@@ -104,30 +113,42 @@ Migration example
 
 - Test that converted code works as expected
 
-- Prepare a Python 2 environment containing Zope 4 (latest), all relevant
-  applications for your ZODB as well as ``zodbupdate`` and ``zodb.py3migrate``
+- Prepare a Python 3 environment, containing:
+
+  - Zope 4 (latest),
+  - all relevant applications and addons for your ZODB,
+  - `zodbupdate <https://pypi.org/project/zodbupdate/>`_,
+  - `zodbverify <https://pypi.org/project/zodbverify/>`_,
+
+- Prepare a Zope configuration
 
   - Create a new Zope instance using ``mkwsgiinstance``
 
   - Update configuration in ``zope.ini`` and ``zope.conf`` to match
     previous Zope2 instance configuration.
 
-  - Run ``zodb-py3migrate-analyze Data.fs`` to determine if third party
-    products have serialized objects into the ZODB that would cause decoding
-    errors in Python 3.
-    Note: This functionality might be rolled into zodbupdate, see https://github.com/zopefoundation/zodbupdate/issues/10.
+- Migrate the database:
+
+  - Make sure no zope instance is running.
+
+  - Dry-run the migration with
+    ``zodbupdate --pack --convert-py3 --dry-run path/to/Data.fs``.
+    This may take a while.
+
+  - If no errors are shown, start the in-place migration of the ZODB
+    ``zodbupdate --pack --convert-py3 path/to/Data.fs``.
+    This may take a while.
 
-  - Run ``zodbupdate --pack --convert-py3 --file Data.fs`` to migrate the ZODB.
+- Check the migrated database:
 
-- Prepare a Python 3 environment with Zope 4 (latest) and all relevant
-  applications.
+  - Verify th ZODB by iterative loading every pickle using
+    ``zodbverify --zodbfile path/to/Data.fs``.
 
   - Start the Application using ``runwsgi etc/zope.ini``.
-  
-  - ``Data.fs.index`` will be discarded at the first start, you can ignore
+    ``Data.fs.index`` will be discarded at the first start, you can ignore
     the error message telling that it cannot be read.
 
-  - Delete and recreate all ZCatalog/Index objects.
+  - If any ZCatalog is in the ZODB, do a clear and rebuild for all of them.
```

Comment:
I think this step is optional and should be done if there are problems regarding the catalog entries. (I did not encounter such problems when migrating a union.cms ZODB.)